Flow.Launcher/ViewModel/MainViewModel.cs

```diff
@@ -813,7 +806,7 @@ public void UpdateResultView(IEnumerable<ResultsForUpdate> resultsForUpdates)
                     else
                     {
                         var priorityScore = metaResults.Metadata.Priority * 150;
-                        result.Score += _userSelectedRecord.GetSelectedCount(result) * 5 + priorityScore;
+                        result.Score += _userSelectedRecord.GetSelectedCount(result) + priorityScore;
```

Comment:
This *5 has been changed to UserSelectedRecords. Old one will get 1 score everytime selected